tests/integration/kubernetes/gha-run.sh

```diff
@@ -20,6 +20,7 @@ KATA_DEPLOY_WAIT_TIMEOUT=${KATA_DEPLOY_WAIT_TIMEOUT:-10m}
 KATA_HYPERVISOR=${KATA_HYPERVISOR:-qemu}
 KUBERNETES="${KUBERNETES:-}"
 SNAPSHOTTER="${SNAPSHOTTER:-}"
+export TEST_CLUSTER_NAMESPACE="${TEST_CLUSTER_NAMESPACE:-kata-containers-k8s-tests}"
```

Comment:
hi @danmihai1 !

Could you define `TEST_CLUSTER_NAMESPACE` in tests/gha-run-k8s-common.sh along-side the *_namespace() functions so that the callers don't need to export it?

Reply:
@wainersm, I tried, but looks like we have to go back to the exported TEST_CLUSTER_NAMESPACE, because tests_common.sh is using that variable too (https://github.com/kata-containers/kata-containers/actions/runs/7809105784/job/21320896440?pr=8922). Unless you think it's worth doing a different type of refactoring here.